package.json

```diff
@@ -12,9 +12,10 @@
     "@testing-library/user-event": "^7.2.1",
     "babel-jest": "^27.0.0",
     "bootstrap": "^4.6.2",
+    "fetch-mock": "^9.11.0",
```

Comment:
note for next time: please double check any added dependencies if they're being used

this dependency is not being used so it should be removed from `package.json` and `package-lock.json` but I'll clean this up once all PRs are merged